processor/src/system/tests.rs

```diff
@@ -9,13 +9,14 @@ fn cycles_num_exceeded() {
     let mut host = DefaultHost::default();
     let program = &MastForest::default();
 
+    let max_cycles = 2048;
```

Comment:
Was this change necessary?

Reply:
Because of some issues/bugs with our temporary simple FRI. There is an issue for this in `0xMiden/Plonky3`

Reply:
https://github.com/0xMiden/miden-vm/issues/2486